src/status_im/utils/publisher.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.utils.publisher
+  (:require [re-frame.core :as re-frame]
+            [re-frame.db]
+            [status-im.constants :as constants]
+            [status-im.accounts.update.publisher :as accounts]
+            [status-im.pairing.core :as pairing]
+            [status-im.utils.async :as async-util]
+            [status-im.data-store.accounts :as accounts-store]
+            [status-im.utils.datetime :as datetime]
+            [status-im.transport.message.protocol :as protocol]
+            [status-im.transport.message.contact :as message.contact]
+            [status-im.transport.shh :as shh]
```

Comment:
`shh`, `message.contact`, `protocol`, `accounts-store`, `constants` are not used